bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix(managed-kafka-cr): ensure that deprecated bf2.org/id and bf2.org/placementId are always present

## Description


Closes https://issues.redhat.com/browse/MGDSTRM-3746

This follows on #331 to make sure that the change remains backward compatible

See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/149
for additional references

/cc @wei-lee @davidffrench @k-wall @shawkins 

## Verification Steps


See the change on the generated `pkg/api/private/openapi/model_managed_kafka_all_of_metadata_annotations.go` and make sure that `omitempty` is not there in the `json` annotation.

## Type of change

- [ ] Refactor (A code change that neither fixes a bug nor adds a feature. The work is done to address technical debt) 

## Checklist (Definition of Done)
